common/src/main/java/mtr/data/Train.java

```diff
@@ -122,7 +125,45 @@ public Train(FriendlyByteBuf packet) {
 			ridingEntities.add(packet.readUUID());
 		}
 
-		inventory = null;
+		inventory = null; // TODO: Verify with Jonathan
```

Comment:
@zbx1425 Inventory is not synced to the client (at least for now). Maybe if we wanna add item rendering inside the train, it would be kinda cool and we will have to sync the inventory to the client.